k-distribution/tests/regression-new/concrete-function/2.a.out

```diff
@@ -1,3 +1,3 @@
 <k>
-  sym2 ( 222 ) +Num 1
+  foo ( sym2 ( 222 ) +Num 1 ) ~> .
```

Comment:
Is this correct? I see there is a rule for `foo`.

Reply:
That rule doesn't apply because it can't prove that +Num is defined on those arguments.